app/wwwroot/js/coa.js

```diff
@@ -188,3 +194,30 @@ $(document).ready(function () {
         $("#btnSelectDate").show();
     });
 });
+
+/**
+ * Handle "Application types" selection change.
+ *
+ * @param {Array} selection - Selected application types
+ */
+function onApplicationTypeChange(selection) {
+    // if any of the selected application types require a full hour,
+    // disable the half hour duration button
+    const hourNeeded = selection.some(({ timeline }) => timeline === "One Hour");
+
+    $halfHourRadio = $('input[name="IsHalfHour"][value="true"]');
+    $halfHourLabel = $halfHourRadio.parent("label");
+
+    if (hourNeeded) {
+        // deselect radio button
+        $halfHourRadio.prop("checked", false);
+        $halfHourLabel.toggleClass("active", false);
+    }
+
+    // disable radio button
+    $halfHourRadio.prop("disabled", hourNeeded);
```

Comment:
We need to talk to Winnie about the messaging and behaviour

Reply:
I'll reach out on Slack shortly!

Reply:
We came up with some fairly simple changes, so I'll implement that shortly and then mark this PR as "ready for review" 👍